packages/app-distribution/lib/modular/index.d.ts

```diff
@@ -0,0 +1,35 @@
+import { ReactNativeFirebase } from '@react-native-firebase/app';
+import { FirebaseAppDistributionTypes } from '..';
+
+export type AppDistribution = FirebaseAppDistributionTypes.Module;
+
+/**
+ * Get an App Distribution instance for the specified app or current app.
+ */
+export declare function getAppDistribution(app?: ReactNativeFirebase.FirebaseApp): AppDistribution;
```

Comment:
Don't we want these all to be `FirebaseApp` types? 🤔

```suggestion
export declare function getAppDistribution(app?: FirebaseApp): AppDistribution;
```